runtime/src/lib.rs

```diff
@@ -372,6 +377,59 @@ impl versioned_store_permissions::Trait for Runtime {
         integration::versioned_store_permissions::ContentLeadOrSudoKeyCanCreateClasses;
 }
 
+type EntityId = <Runtime as content_directory::Trait>::EntityId;
+
+parameter_types! {
+    pub const PropertyNameLengthConstraint: InputValidationLengthConstraint = InputValidationLengthConstraint::new(1, 49);
+    pub const PropertyDescriptionLengthConstraint: InputValidationLengthConstraint = InputValidationLengthConstraint::new(1, 500);
+    pub const ClassNameLengthConstraint: InputValidationLengthConstraint = InputValidationLengthConstraint::new(1, 49);
+    pub const ClassDescriptionLengthConstraint: InputValidationLengthConstraint = InputValidationLengthConstraint::new(1, 500);
+
```

Comment:
Do we need spaces between lines?

Reply:
The purpose is to make constraints of different types distinguishable. I can remove them, though.

Reply:
Yes, please. Remove or introduce explicit comments for different types.

Reply:
Done. Removed them in content directory `mock.rs` as well. https://github.com/Joystream/joystream/pull/1273/commits/d993a64a39c491aad16e4c3d0f1b041496d41f37